pkg/util/deployment/deployment.go

```diff
@@ -28,8 +28,13 @@ import (
 	podutil "k8s.io/kubernetes/pkg/util/pod"
 )
 
+const (
+	// The revision annotation of a deployment's replication controllers which records its rollout sequence
+	RevisionAnnotation = "deployment.kubernetes.io/revision"
```

Comment:
I agree that all the deployment-related annotations belong in the same place, but it's ok with me if this gets moved in a subsequent PR, since we have so many interdependent PRs right now.